redo.dev/src/components/demos/model.tsx

```diff
@@ -1,25 +1,43 @@
 import { Box } from "@mui/material"
-import React from "react"
+import model from "raw-loader!/stackblitz/generated/model.ts.raw"
```

Comment:
We probably need to import all of the raw demo files here and use them to create something like  `contentsByEmbedId: Record<EmbedId, string>`.